src/hw/usb.h

```diff
@@ -78,7 +78,7 @@ struct usbhub_op_s {
  ****************************************************************/
 
 // USB mandated timings (in ms)
-#define USB_TIME_SIGATT 500 // 100		// WIV changed to 500 to make ADATA CIO3 stick working ( 200 is sufficient for SanDisk Ultra USB 3.0)
+#define USB_TIME_SIGATT 1000 // 100		// WIV changed to 500 to make ADATA CIO3 stick working ( 200 is sufficient for SanDisk Ultra USB 3.0)
 #define USB_TIME_ATTDB  100
 #define USB_TIME_DRST   10
```

Comment:
@miczyg1 please change comment for this entry according to recent reasoning.

Reply:
@pietrushnic done